lightkurve/correctors/regressioncorrector.py

```diff
@@ -209,6 +209,9 @@ def correct(self, design_matrix_collection, cadence_mask=None, sigma=5,
         design_matrix_collection.validate()
         self.design_matrix_collection = design_matrix_collection
 
+        # Validate the design matrix. Emits a warning if the matrix has low rank.
+        self.design_matrix_collection.validate()
```

Comment:
This seems identical to what happens on line 209 -- is it necessary?

Reply:
🤦🏻‍♂️ whoops, that is unnecessary. Fixed.